packages/node-opentelemetry/README.md

```diff
@@ -113,7 +113,30 @@ And run your application with the following command (example using `ts-node`):
 npx ts-node -r './instrument.ts' index.ts
 ```
 
-### Advanced Instrumentation Configuration (Optional)
+### (Optional) Attach User Information or Metadata
+
+Attaching user information will allow you to search/filter sessions and events in HyperDX.
```

Comment:
Thoughts on this replacing lines 118-120?

```
To easily tag all events related to a given attribute or identifier (ex. user id or email), you can call the `setTraceAttributes` function which will tag every log/span associated with the current trace after the call with the declared attributes. It's recommended to call this function as early as possible within a given request/trace (ex. as early in an Express middleware stack as possible).

This is a convenient way to ensure all logs/spans are automatically tagged with the right identifiers to be searched on later, instead of needing to manually tagging and propagating identifiers yourself.
```